pkg/internal/codegen/parse/util.go

```diff
@@ -395,3 +411,96 @@ func parseScaleParams(t *types.Type) (map[string]string, error) {
 	}
 	return nil, fmt.Errorf(jsonPathError)
 }
+
+// PrintColumnKV parses key-value string formatted as "foo=bar" and returns key and value.
+func PrintColumnKV(s string) (key, value string, err error) {
```

Comment:
May be rename it to `parseKV` because I am assuming this will removed once the utils pkg is available to be used here ? Also, add a TODO with comment this is duplicated and will be removed in a followup PR.

Reply:
I will include this feature in a separate PR if that's okay with you.

Reply:
>  Also, add a TODO with comment this is duplicated and will be removed in a followup PR.

This function is not duplicated, as I said before I made changes in the logic `kv := strings.SplitN(s, "=", 2)` to support test cases for `JSONPath` printcolumn which are already listed in the test file.

Reply:
ok. In current form, this is being exported (public function). Lets rename (lowercase) it to `parsePrintColumnKV` or `printColumnKV` to make it private.